packages/pds/src/config.ts

```diff
@@ -50,6 +50,8 @@ export interface ServerConfigValues {
   hiveApiKey?: string
   labelerDid: string
   labelerKeywords: Record<string, string>
+  unacceptableHandleWordsB64?: string
```

Comment:
nitpick: Loading this file would be better for transparency. Keep the env var though, the file should be a fallback used in prod!

Reply:
yeah I think it'll be great if we know what are the words that are considered as unacceptable words here for transparency

Reply:
Not just for transparency sake, but for future instances as well. A list contained solely in an env var means no other created instances get access